webpack.config.js

```diff
@@ -1,52 +1,38 @@
 const
   webpack = require('webpack'),
-  path = require('path'),
   version = require('./package.json').version;
 
 module.exports = {
+  name: 'browser',
   mode: 'production',
   entry: './index.js',
   output: {
     path: `${__dirname}/dist`,
     filename: 'kuzzle.js',
-    library: 'KuzzleSDK',
+    library: 'kuzzle-sdk',
```

Comment:
The value of the `library` attribute should be a valid Javascipt reference name (i.e. dashes will cause trouble). See https://webpack.js.org/configuration/output/#output-library

> The variable MyLibrary will be bound with the return value of your entry file, if the resulting output is included as a script tag in an HTML page.